### PR TITLE
adding filter section and upgrade to pyclarify version 0.3.5

### PR DESCRIPTION
**Pull request by oddaspa**
NB! Did not touch the Orchest Pipeline Tutorial.

### CODE REVIEW

File: tutorials/Forecasting.ipynb [hunk 1]
```diff
@@ -167,52 +163,39 @@
...
-        "        ]\n",
-        "      }\n",
-        "    }\n",
-        "  },\n",
-        "  \"data\": {\n",
-        "    \"include\": True,\n",
-        "    \"notBefore\": starting_date #starting from \n",
-        "  }\n",
-        "}\n",
+        "item_id = \"c5jdi4jsbu8cohpla9vg\" #change for the you item_id\n",
```
Comment (AlexiaArtemis) [inline tutorials/Forecasting.ipynb:166]: Maybe have it as <item_id> instead if the actuall id, with an example. The same with the starting_date.

File: tutorials/Introduction.ipynb [hunk 2]
```diff
@@ -77,21 +77,22 @@
...
-        "outputId": "a24f128d-82d1-4623-d36e-074dd72aeeb9"
+        "id": "d2f69f4e",
+        "vscode": {
+          "languageId": "python"
+        }
       },
       "outputs": [],
       "source": [
-        "!pip install pyclarify"
+        "!pip install -U pyclarify"
```
Comment (AlexiaArtemis) [inline tutorials/Introduction.ipynb:87]: What is the use of -U ?

**Review by AlexiaArtemis — APPROVED** [review]
Great work! 🙌

File: tutorials/Introduction.ipynb (hunk 2 above)
Comment (oddaspa) [reply tutorials/Introduction.ipynb:87]: Its shorthand for `--upgrade`, I used it when testing, will remove :)

**Pull request merged**